relayer/chains/evm/listener.go

```diff
@@ -53,33 +53,29 @@ func (p *Provider) Listener(ctx context.Context, lastSavedHeight uint64, blockIn
 
 	var (
 		subscribeStart = time.NewTicker(time.Second * 1)
-		isSubError     bool
 		latestHeight   = p.latestHeight(ctx)
 		concurrency    = p.GetConcurrency(ctx, startHeight, latestHeight)
```

Comment:
concurrency is initialized only once in this function, but I think it should be recalculated every time before you start polling.

Reply:
yep, nice one.